packages/bpk-component-page-indicator/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "bpk-component-page-indicator",
+  "version": "2.1.1",
+  "description": "Backpack page-indicator component.",
+  "license": "Apache-2.0",
+  "private": true,
```

Comment:
Can you also remove this line?

(These arent your fault but more out automation not updating when creating new components)

Reply:
Have remove this line, thank you!